examples/with-fauna/pages/index.js

```diff
@@ -1,184 +1,183 @@
-import Head from 'next/head'
-import { useState } from 'react'
-import cn from 'classnames'
-import formatDate from 'date-fns/format'
-import useSWR, { mutate } from 'swr'
-import 'tailwindcss/tailwind.css'
-import { listGuestbookEntries } from '@/lib/fauna'
-import SuccessMessage from '@/components/SuccessMessage'
-import ErrorMessage from '@/components/ErrorMessage'
-import LoadingSpinner from '@/components/LoadingSpinner'
-
-const ENTRIES_PATH = '/api/entries'
+import Head from "next/head";
+import { useState } from "react";
+import cn from "classnames";
+import formatDate from "date-fns/format";
+import useSWR, { mutate, SWRConfig } from "swr";
+import "tailwindcss/tailwind.css";
+import { listGuestbookEntries } from "@/lib/fauna";
+import SuccessMessage from "@/components/SuccessMessage";
+import ErrorMessage from "@/components/ErrorMessage";
+import LoadingSpinner from "@/components/LoadingSpinner";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
 
 const putEntry = (payload) =>
-  fetch(ENTRIES_PATH, {
-    method: 'POST',
-    body: JSON.stringify(payload),
-    headers: {
-      'Content-Type': 'application/json',
-    },
-  }).then((res) => (res.ok ? res.json() : Promise.reject(res)))
-
-const useEntriesFlow = ({ initialEntries }) => {
-  const { data: entries } = useSWR(ENTRIES_PATH, {
-    initialData: initialEntries,
-  })
-
-  const onSubmit = async (payload) => {
-    await putEntry(payload)
-    await mutate(ENTRIES_PATH)
-  }
-
-  return {
-    entries,
-    onSubmit,
-  }
-}
+	fetch("/api/entries", {
```

Comment:
Could you run the formatting & linting scripts? 

`pnpm run prettier-fix && pnpm run lint-fix`